_std/defines/obj.dm

```diff
@@ -11,3 +11,5 @@
 /// At which alpha do opague objects become see-through?
 #define MATERIAL_ALPHA_OPACITY 190
 
+/// Overrides multicontext menu popup when an object is clicked by a player who is holding an item
+#define IGNORE_CONTEXT_CLICK_HELD 200
```

Comment:
uhh what? this isn't how flags work

Reply:
how would you suggest i change this?

Reply:
to use a power of two, like is necessary for binary-based flag systems

Reply:
see: the other flags above this snippet

Reply:
it should be fixed unless i'm awful at math